pkgs/by-name/wo/world-wall-clock/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "world-wall-clock";
+  version = "0.1.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ddelabru";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
This is now the preferred argument when fetching a tag. `rev` should only be used for specific commits.

---

Changed `rev` to `tag`.